src/common/cmd.c

```diff
@@ -493,3 +544,20 @@ static cmdstat_t cmd_wait_fn(i32 argc, const char** argv)
         return cmdstat_ok;
     }
 }
+
+static cmdstat_t cmd_cmds_fn(i32 argc, const char** argv)
+{
+    Con_Logf(LogSev_Info, "cmd", "displaying %i commands (`help <cmd>` for more info):", ms_cmds.count);
+
+    u32* indices = StrDict_Sort(&ms_cmds, SDictStrCmp, NULL);
+
+    cmddef_t def = { 0 };
+    for (u32 i = 0; i < ms_cmds.count; i++)
+    {
+        u32 index = indices[i];
+        const char* key = ms_cmds.keys[index];
+        Con_Logf(LogSev_Info, "cmd", "\t%s", key);
+    }
```

Comment:
I was not sure how to iterate over the dict any other way. Is this the best way?

Reply:
This is a good way to do it.
You can also loop over ms_cmds.keys by ms_cmds.width and check if ms_cmds.keys[i] is not null, but it won't be sorted.